lnrpc/invoicesrpc/invoices_server.go

```diff
@@ -274,6 +274,7 @@ func (s *Server) AddHoldInvoice(ctx context.Context,
 		FallbackAddr:    invoice.FallbackAddr,
 		CltvExpiry:      invoice.CltvExpiry,
 		Private:         invoice.Private,
+		HodlInvoice:     true,
```

Comment:
would be nice to explicitly initialize `PaymentPreimage` to nil here

Reply:
Done